axios-api-react/src/components/ActivityList.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { connect } from "react-redux";
+import { fetchBored } from "../actions/index";
+import { useDispatch } from 'react-redux';
+import ActivityCard from './ActivityCard';
+
+function ActivityList(props) {
+    console.log("ActivityList", props);
+    const dispatch = useDispatch();
+    return (
+        <>
+            <button onClick={() => dispatch(fetchBored())}>Get Pictures</button>
+            {props.isFetching && <span role="img" alt="clock">⏰</span>}
+            {props.error && <div>{props.error.message}</div>}
+            {props.activity &&
+                props.activity.map(activity =>
+                    <ActivityCard className="activity" key={activity.url} ActivityList={activity} />)}
+
```

Comment:
This would be a good area to start writing some comments to explain the logic and goals